dll/win32/shell32/CDefView.cpp

```diff
@@ -1398,10 +1398,10 @@ HRESULT CDefView::InvokeContextMenuCommand(UINT uCommand)
     cmi.lpVerb = MAKEINTRESOURCEA(uCommand);
     cmi.hwnd = m_hWnd;
 
-    if (GetKeyState(VK_SHIFT) & 0x8000)
+    if (GetAsyncKeyState(VK_SHIFT) < 0)
```

Comment:
I have removed these controversial changes for now in hopes that this can get committed more easily. The remaining changes fix about 90% of what I was trying to accomplish by mostly fixing the "shift-delete" file problem. I plan to address the remaining issues in another Jira ticket and PR. Thanks for everyone's help with this. Doug Lyons